help_test.go

```diff
@@ -512,3 +512,55 @@ Group 2
 		require.Equal(t, expected, w.String())
 	})
 }
+
+func TestUsageOnError(t *testing.T) {
+	var cli struct {
+		Flag string `help:"A required flag." required`
+	}
+	w := &strings.Builder{}
+	p := mustNew(t, &cli,
+		kong.Writers(w, w),
+		kong.Description("Some description."),
+		kong.Exit(func(int) {}),
+		kong.UsageOnError(),
+	)
+	_, err := p.Parse([]string{"--help"})
```

Comment:
Shouldn't this be parsing an empty slice to test the error?

Reply:
Yes, sorry. Sloppy. Fixed in fixup commit 891ebfd .